src/tests/Dictionary/DictionaryContainer.test.jsx

```diff
@@ -1,23 +1,32 @@
 import React from 'react';
-import { mount, shallow } from 'enzyme';
-import { DictionaryOverview, mapStateToProps }from '../../components/dashboard/components/dictionary/DictionaryContainer';
-import dictionary from '../__mocks__/dictionaries';
 import { MemoryRouter } from 'react-router-dom';
+import { mount } from 'enzyme';
+import { DictionaryOverview } from '../../components/dashboard/components/dictionary/DictionaryContainer';
+import dictionary from '../__mocks__/dictionaries';
 
 jest.mock('../../components/dashboard/components/dictionary/AddDictionary');
 describe('DictionaryOverview', () => {
   it('should render without any data', () => {
     const props = {
       dictionary: [],
     };
-    const wrapper = <MemoryRouter><DictionaryOverview {...props} /></MemoryRouter>
+    const wrapper = <MemoryRouter><DictionaryOverview {...props} /></MemoryRouter>;
```

Comment:
Make more assertions here, a snapshot is not enough

Reply:
That is a nice suggestion the only problem is this pull request needs to be merged since it a bug fix probably. I could probably take up another task to add enough assertions

Reply:
At least one assertion would be enough.

Reply:
I see.